<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hazama</class>
 <widget class="QMainWindow" name="Hazama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hazama</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="server_mode_radio">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ServerMode</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="algorithmSelector">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Algorithm Selector</string>
    </property>
    <widget class="QRadioButton" name="algo4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>BeamAlgorithmWithSearchSameLength</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="algo3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>BeamAlgorithm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="algo1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>SimpleAlgorithm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="algo2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>MountainAlgorithm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="algo0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>PoorAlgorithm</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="algo5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>StepSearch</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>821</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcdNumber_1"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>174</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="useWebCamera">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UseWebCamera</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="useFileData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UseFileData</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="useImageData">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UseImageData</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectImageData">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SelectImageData</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="LoadConfig">
       <property name="text">
        <string>LoadConfig</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="WriteConfig">
       <property name="text">
        <string>WriteConfig</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="isKeepBestAnswerMode">
       <property name="text">
        <string>KeepBestAnswerMode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="pieceURL">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>110</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>./../../procon2017-comp/sample/sample_pieces_3.JPG</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="franeURL">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>150</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>./../../procon2017-comp/sample/sample_frame_3.JPG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Piece_URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame_URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>320</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>64</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>64</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>64</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>200</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>64</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>useImageData</tabstop>
  <tabstop>server_mode_radio</tabstop>
  <tabstop>algo4</tabstop>
  <tabstop>algo3</tabstop>
  <tabstop>algo1</tabstop>
  <tabstop>algo2</tabstop>
  <tabstop>algo0</tabstop>
  <tabstop>horizontalSlider_1</tabstop>
  <tabstop>horizontalSlider_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>horizontalSlider_4</tabstop>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>horizontalSlider_2</tabstop>
  <tabstop>RunButton</tabstop>
  <tabstop>useWebCamera</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Hazama</receiver>
   <slot>pauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1002</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Hazama</receiver>
   <slot>startClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1012</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>left</sender>
   <signal>clicked()</signal>
   <receiver>Hazama</receiver>
   <slot>leftClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>right</sender>
   <signal>clicked()</signal>
   <receiver>Hazama</receiver>
   <slot>rightClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1084</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>1037</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pauseClicked()</slot>
  <slot>startClicked()</slot>
  <slot>leftClicked()</slot>
  <slot>rightClicked()</slot>
 </slots>
</ui>
